glean-core/src/histogram/functional.rs

```diff
@@ -24,6 +26,8 @@ pub struct Functional {
 impl Functional {
     /// Instantiate a new functional bucketing.
     fn new(log_base: f64, buckets_per_magnitude: f64) -> Functional {
+        let _ = FloatingPointContext::new();
```

Comment:
Assigning to `_` will drop immediately. If you want to drop it at the end of the scope you need to give it a name:

```suggestion
        let _fpc = FloatingPointContext::new();
```